### DISCUSSION TITLE
Clarification of Score Calculation

### DISCUSSION THREAD

**Discussion by vic816** [CastleLab/COMP3021-2021Fall-PA1-Student-Version#29] [Q&A] [answered]
When a move causes the player to die, will the score be deducted by 4 or 5 (one due to a valid move)? 

Also, is the score allowed to be negative or the minimum limit is zero? Thank you.

**Comment by Derppening** [comment, marked as answer]
The Javadoc says:

```java
<li>Each death deducts four points (on top of the one point deducted by a valid move).</li>
```

So each death deducts 4 points *plus* the one point deducted by a valid move.

**Reply by vic816** [reply]
May I know can the score be negative though?

**Reply by Derppening** [reply]
Right sorry, missed that question.

Score can be negative.
